smarthash.py

```diff
@@ -191,35 +192,37 @@
     if len(nfos) > 0 and not imdb_id:
         nfo = nfos[0]
 
-    # manual imdb_id override
-    if args.imdb_id:
-        imdb_id = args.imdb_id
+    if 'imdb-id' in plugins[args.plugin].options:
```

Comment:
Why was this refactored? If the --imdb-id flag is not passed, the code inside already won't run... or am I missing something?

Reply:
This was done because if the plugin doesn't have it in options then args won't necessarily have an imdb_id which could cause a naming error.

In Python 3.8 on my Ubuntu 18.04 system I was getting:
```Traceback (most recent call last):
  File "smarthash.py", line 196, in <module>
    if args.imdb_id:
AttributeError: 'Namespace' object has no attribute 'imdb_id'```

Reply:
Good point.